social_core/backends/okta.py

```diff
@@ -0,0 +1,79 @@
+"""
+Okta OAuth2 and OpenIdConnect:
+    https://python-social-auth.readthedocs.io/en/latest/backends/okta.html
+"""
+import sys
+if sys.version_info[0] < 3:
+    import urlparse as urlparse
+else:
+    import urllib.parse as urlparse
+
+import requests
+from six.moves.urllib.parse import urljoin
+from jose import jwk, jwt, jws
```

Comment:
I do not see the jose imports being use in the code below. Are they necessary ?

Reply:
I've reorganised the imports to be more efficient.